contrib/soju/template.py

```diff
@@ -0,0 +1,65 @@
+pkgname = "soju"
+pkgver = "0.7.0"
+pkgrel = 0
+build_style = "go"
+make_build_args = ["./contrib/...", "./cmd/..."]
+hostmakedepends = ["go", "scdoc"]
+makedepends = ["sqlite-devel"]
+depends = ["ca-certificates", "libcap-progs"]
+go_build_tags = ["libsqlite3"]
+go_check_tags = list(go_build_tags)
+pkgdesc = "IRC bouncer"
+maintainer = "ttyyls <contact@behri.org>"
+license = "AGPL-3.0-or-later"
+url = "https://soju.im"
+source = f"https://git.sr.ht/~emersion/soju/archive/v{pkgver}.tar.gz"
+sha256 = "089b900ccd7fa009b7ae50b9c2b9e77099b50b2499422649c3b27aff7d62d109"
+
+
+def do_build(self):
+    self.golang.build(
+        args=[
+            "-ldflags",
+            "-X 'git.sr.ht/~emersion/soju/config.DefaultPath=/etc/config/soju' \
+	        -X 'git.sr.ht/~emersion/soju/config.DefaultUnixAdminPath=/run/soju/admin'",
+        ],
```

Comment:
you can pass this in make_build_args

Reply:
see tailscale

Reply:
the order wouldn't be right as they'd be put after -o

Reply:
with the previous patch to args, it works as expected

Reply:
you mean the thing you already patched here? or something else?

Reply:
yes the first commit in this pr

Reply:
actually looking again i'm not sure what you're talking about. in tailscale the build line looks like

```
go build -p 16 -v -trimpath -o /builddir/tailscale-1.58.2/build/ -ldflags= -X tailscale.com/version.longStamp=1.58.2.222 -X tailscale.com/version.shortStamp=1.58.2.22222 ./cmd/tailscale ./cmd/tailscaled
```

(all the things are passed after `-o`)

and yet, it works as expected. the ldflags are read fine. what are you talking about?

Reply:
yeah, i just tested removing the patch from here and then  

```
make_build_args = [
    "-ldflags="
    " -X git.sr.ht/~emersion/soju/config.DefaultPath=/etc/config/soju2"
    " -X git.sr.ht/~emersion/soju/config.DefaultUnixAdminPath=/run/soju/admin",
    "./contrib/...",
    "./cmd/...",
]
```

and this works completely fine, so i'm confused what you were trying to fix

Reply:
also the go build command disagrees with you:  

```
$ go build
usage: go build [-o output] [build flags] [packages]
```

everything is supposed to come after `-o`. not before it

Reply:
you're right. failure of those tests must've been due to sth unrelated to it.

Reply:
> also the go build command disagrees with you:
just dropped it